nonebot_bison.platform.platform.CategoryNotSupport报错

### nonebot-bison 版本

v0.9.6-rc2

### nonebot 版本

直接拉取的docker镜像

### 安装方式

None

### 操作系统

debian12

### 问题

获取动态时总会弹出如下报错

### 日志

```shell
nonebot-1  | 01-20 17:05:54 [INFO] nonebot_bison | fetch new post from bilibili 35040323: 1024458450817515522
nonebot-1  | 01-20 17:05:54 [WARNING] nonebot_bison | API request record: https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/space?host_mid=35040323&timezone_offset=-480&offset=&features=itemOpusStyle Headers({'host': 'api.bilibili.com', 'accept': '*/*', 'accept-encoding': 'gzip, deflate, br', 'connection': 'keep-alive', 'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0', 'cookie': | [200] Headers({'date': 'Mon, 20 Jan 2025 09:05:54 GMT', 'content-type': 'application/json; charset=utf-8', 'transfer-encoding': 'chunked', 'connection': 'keep-alive', 'bili-status-code': '0', 'bili-trace-id': '189a7d82e8678e11', 'cpu_usage': '60', 'x-bili-trace-id': '369a526c3804e7e5189a7d82e8678e11', 'x-rid-result': '2', 'content-encoding': 'gzip', 'expires': 'Mon, 20 Jan 2025 09:05:53 GMT', 'cache-control': 'no-cache', 'x-cache-webcdn': 'BYPASS from blzone02'}) {"code":0,"message":"0","ttl":1,"data":{"has_more":true,"items":[{"basic":{"comment_id_str":"314669956","comment_type":11,"jump_url":"//www.bilibili.com/opus/924682449980489833","like_icon":{"action_url":"","end_url":"","id":0,"start_url":""},"rid_str":"314669956"},"id_str":"924682449980489833"　后略......

nonebot-1  | 01-20 17:05:54 [ERROR] apscheduler | Job "Scheduler.exec_fetch (trigger: interval[0:01:00], next run at: 2025-01-20 17:06:54 CST)" raised an exception
nonebot-1  | Traceback (most recent call last):
nonebot-1  |   File "<string>", line 25, in <module>
nonebot-1  |   File "/app/.venv/lib/python3.12/site-packages/nonebot/__init__.py", line 337, in run
nonebot-1  |     get_driver().run(*args, **kwargs)
nonebot-1  |   File "/app/.venv/lib/python3.12/site-packages/nonebot/drivers/fastapi.py", line 188, in run
nonebot-1  |     uvicorn.run(
nonebot-1  |   File "/app/.venv/lib/python3.12/site-packages/uvicorn/main.py", line 579, in run
nonebot-1  |     server.run()
nonebot-1  |   File "/app/.venv/lib/python3.12/site-packages/uvicorn/server.py", line 66, in run
nonebot-1  |     return asyncio.run(self.serve(sockets=sockets))
nonebot-1  |   File "/usr/local/lib/python3.12/asyncio/runners.py", line 194, in run
nonebot-1  |     return runner.run(main)
nonebot-1  |   File "/usr/local/lib/python3.12/asyncio/runners.py", line 118, in run
nonebot-1  |     return self._loop.run_until_complete(task)
nonebot-1  | > File "/app/.venv/lib/python3.12/site-packages/apscheduler/executors/base.py", line 181, in run_coroutine_job
nonebot-1  |     retval = await job.func(*job.args, **job.kwargs)
nonebot-1  |   File "/app/nonebot_bison/scheduler/scheduler.py", line 108, in exec_fetch
nonebot-1  |     to_send = await platform_obj.do_fetch_new_post(SubUnit(schedulable.target, send_userinfo_list))
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 108, in do_fetch_new_post
nonebot-1  |     return await catch_network_error(self.fetch_new_post, sub_unit) or []
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 53, in catch_network_error
nonebot-1  |     return await func(*args, **kwargs)
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 343, in fetch_new_post
nonebot-1  |     return await self._handle_new_post(post_list, sub_unit)
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 337, in _handle_new_post
nonebot-1  |     res = await self.dispatch_user_post(new_posts, sub_unit)
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 213, in dispatch_user_post
nonebot-1  |     user_post.append(await self.do_parse(raw_post))
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 245, in do_parse
nonebot-1  |     raise err
nonebot-1  |   File "/app/nonebot_bison/platform/platform.py", line 240, in do_parse
nonebot-1  |     self.parse_cache[post_id] = await self.parse(raw_post)
nonebot-1  |   File "/app/nonebot_bison/platform/bilibili/platforms.py", line 267, in parse
nonebot-1  |     parsed_raw_post = self.pre_parse_by_mojar(raw_post)
nonebot-1  |   File "/app/nonebot_bison/platform/bilibili/platforms.py", line 255, in pre_parse_by_mojar
nonebot-1  |     raise CategoryNotSupport(unknown_type)
nonebot-1  | nonebot_bison.platform.platform.CategoryNotSupport: ('MAJOR_TYPE_OPUS', ['https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/space?host_mid=35040323&timezone_offset=-480&offset=&features=itemOpusStyle Headers({\'host\': \'api.bilibili.com\', \'accept\': \'*/*\', \'accept-encoding\': \'gzip, deflate, br\', \'connection\': \'keep-alive\', \'user-agent\': \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0\',  | [200] Headers({\'date\': \'Mon, 20 Jan 2025 09:05:54 GMT\', \'content-type\': \'application/json; charset=utf-8\', \'transfer-encoding\': \'chunked\', \'connection\': \'keep-alive\', \'bili-status-code\': \'0\', \'bili-trace-id\': \'189a7d82e8678e11\', \'cpu_usage\': \'60\', \'x-bili-trace-id\': \'369a526c3804e7e5189a7d82e8678e11\', \'x-rid-result\': \'2\', \'content-encoding\': \'gzip\', \'expires\': \'Mon, 20 Jan 2025 09:05:53 GMT\', \'cache-control\': \'no-cache\', \'x-cache-webcdn\': \'BYPASS from blzone02\'}) {"code":0,"message":"0","ttl":1,"data":{"has_more":true,"items":[{"basic":{"comment_id_str":"314669956","comment_type":11,"jump_url":"//www.bilibili.com/opus/924682449980489833","like_icon":{"action_url":"","end_url":"","id":0,"start_url":""},"rid_str":"314669956"},"id_str":"924682449980489833","modules":{"module_author":{"avatar":{"container_size":{"height":1.5,"width":1.5},"fallback_layers":{"is_critical_group":true,"layers":[{"general_spec":{"pos_spec":{"axis_x":0.75,"axis_y":0.75,"coordinate_pos":2},"render_spec":{"opacity":1},"size_spec":{"height":1.3,"width":1.3}},"layer_config":{"is_critical":true,"tags":{"AVATAR_LAYER":{},"GENERAL_CFG":{"config_type":1,"general_config":{"web_css_style":{"borderRadius":"50%"}}}}},"resource":{"res_image":{"image_src":{"placeholder":6,"remote":{"bfs_style":"widget-layer-avatar","url":"https://i2.hdslb.com/bfs/face/28580871c60a52968b5412080f0e588ba018e443.jpg"},"src_type":1}},"res_type":3},"visible":true},{"general_spec":{"pos_spec":{"axis_x":1.025,"axis_y":1.025,"coordinate_pos":1},"render_spec":{"opacity":1},"size_spec":{"height":0.41666666666666663,"width":0.41666666666666663}},"layer_config":{"tags":{"GENERAL_CFG":{"config_type":1,"general_config":{"web_css_style":{"background-color":"rgb(255,255,255)","border":"2px solid rgba(255,255,255,1)","borderRadius":"50%","boxSizing":"border-box"}}},"ICON_LAYER":{}}},"resource":{"res_animation":{"webp_src":{"placeholder":5,"remote":{"bfs_style":"widget-layer-avatar","url":"https://i0.hdslb.com/bfs/activity-plat/static/20220506/334553dd7c506a92b88eaf4d59ac8b4d/j8AeXAkEul.gif"},"src_type":1}},"res_type":4},"visible":true}]},"mid":"35040323"},"decorate":{"card_url":"https://i0.hdslb.com/bfs/garb/62001a2090b5760c38e87ce8b0ff2cfc13a6d44c.png","fan":{"color":"#EA618F","color_format":{"colors":["#EA618FFF","#EA618FFF"],"end_point":"0,100","gradients":[0,100],"start_point":"0,0"},"is_fan":true,"num_prefix":"NO.","num_str":"001112","number":1112},"id":1733741236002,"jump_url":"https://www.bilibili.com/h5/mall/equity-link/collect-home?item_id=1733741236002\\u0026isdiy=0\\u0026part=card\\u0026from=post\\u0026f_source=garb\\u0026vmid=35040323\\u0026native.theme=1\\u0026navhide=1","name":"孤独摇滚!粉丝"," 后略......
```

### 确认

- [x] 我搜索过了 issue，但是并没有发现过与我类似的问题
- [x] 我确认在日志中去掉了敏感信息

--- a/nonebot_bison/platform/bilibili/models.py
+++ b/nonebot_bison/platform/bilibili/models.py
@@ -323,7 +323,7 @@ class OPUSMajor(Base):
         pics: "list[OPUSMajor.Pic]"
 
     type: Literal["MAJOR_TYPE_OPUS"]
-    opus: "OPUSMajor.Opus"
+    opus: "list[OPUSMajor.Opus]"
 
 
 class CommonMajor(Base):

